apps/openmw/mwgui/inventorywindow.cpp

```diff
@@ -228,9 +225,9 @@ namespace MWGui
         std::string sound = item.mBase.getClass().getDownSoundId(item.mBase);
 
         MWWorld::Ptr object = item.mBase;
+        bool shift = MyGUI::InputManager::getInstance().isShiftPressed();
```

Comment:
Why has this line been repositioned? Usually, you want to avoid changing anything you don't have to as it reduces the chance of merge conflicts.

Reply:
I was editing this method at first, looks like i forgot to revert this line